src/averageAges.js

```diff
@@ -37,7 +51,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const mothersNames = people
+    .filter(person => person.mother !== null)
```

Comment:
```suggestion
    .filter(person => person.mother)
```
If you want to filter persons where the mother exists, you can just return like this instead of comparing with null. It will handle all falsy values.